vita3k/cpu/src/unicorn_cpu.cpp

```diff
@@ -364,7 +374,7 @@ CPUContext UnicornCPU::save_context() {
     ctx.set_pc(is_thumb_mode() ? get_pc() | 1 : get_pc());
 
     for (size_t i = 0; i < ctx.fpu_registers.size(); i++) {
-        ctx.fpu_registers[i] = get_float_reg(i);
+        ctx.fpu_registers[i] = float_to_binary(get_float_reg(i));
```

Comment:
Nice catch. You can just do reinterpret_cast<uint32_t&>(float) and reinterpret_cast<float&>(uint32_t)

Reply:
I can't convert function result with reinterpret_cast. :(

Reply:
actually better way is just change type of fpu_registers to array of float

Reply:
it's our type

Reply:
https://github.com/Vita3K/Vita3K/blob/168dfb19caa42c1216152e11bbc209ec7d2b8cb0/vita3k/cpu/src/dynarmic_cpu.cpp#L411
here you can just use memcpy to copy it directly. Bit pattern should be exactly same.

Reply:
memcpy(&ctx.fpu_registers, dctx.ExtRegs().data(), sizeof(ctx.fpu_registers));